src/entity/Keyword.ts

```diff
@@ -8,7 +8,14 @@ import {
 
 export type KeywordStatus = 'pending' | 'allow' | 'deny' | 'synonym';
 
+export type KeywordFlags = Partial<{
+  onboarding: boolean;
+}>;
+
+export type KeywordFlagsPublic = never;
+
 @Entity()
+@Index(['value', 'status'])
```

Comment:
Not sure if you want to add a name to it perhaps? (optional)

Reply:
I don't see we name them usually, name is also handled automatically in that case by typeorm which I think is better honestly?

Reply:
https://github.com/dailydotdev/daily-api/pull/1481#discussion_r1365003145 I just seen this discussion 😄 Let's see.

Reply:
I'll add.

Reply:
Why do we need this index?

Reply:
Due to this https://github.com/dailydotdev/daily-api/pull/1480#discussion_r1362163808

Reply:
Oh it's because of the order by. So the right order is first status and then value. Where clause terms should be before order by terms afaik

Reply:
It's not due to ORDER only, status and value are used in WHERE. But since psql used left to right principle it would make sense to put status first yes.

Reply:
Added in 053c24c744860b38df07347bfbe8b0f014f2d74a